tests/chat_completions_detection.rs

```diff
@@ -163,15 +160,14 @@ async fn no_detections() -> Result<(), anyhow::Error> {
     // Make orchestrator call for input/output no detections
     let response = orchestrator_server
         .post(ORCHESTRATOR_CHAT_COMPLETIONS_DETECTION_ENDPOINT)
-        .json(&ChatCompletionsRequest {
-            model: MODEL_ID.into(),
-            detectors: Some(DetectorConfig {
+        .json(&json!({
+            "model": MODEL_ID,
+            "detectors": DetectorConfig {
                 input: HashMap::from([(detector_name.into(), DetectorParams::new())]),
                 output: HashMap::from([(detector_name.into(), DetectorParams::new())]),
-            }),
-            messages,
-            ..Default::default()
-        })
+            },
+            "messages": messages,
+        }))
```

Comment:
Unless you see a benefit on leaving these structs, if you're going to use `json!()` I'd suggest making it look more like an actual json to make it easier to read.
```suggestion
        .json(&json!({
            "model": MODEL_ID,
            "detectors": {
                "input": {
                    detector_name: {}
                },
                "output": {
                    detector_name: {}
                }
            },
            "messages": messages,
        }))
```

Reply:
If you agree with this, please do the same for all other tests you've changed.

Reply:
Either way works, the macro implicitly converts the string or Rust type into `serde_json::Value` and it was faster to just update the existing fields to strings, leaving the values as-is. Will update.